kwok/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start from a base image
+FROM registry.access.redhat.com/ubi8:latest
+
+# Update packages
+RUN yum update -y && yum clean all
+
+RUN yum install -y bash
+
+# Install curl and other dependencies
+RUN yum install -y curl tar && yum clean all
```

Comment:
all those tools are probably already in the UBI image as well.

Reply:
Updated